catalog/spatial/registry/registry-rest-endpoint/src/main/resources/templates/report.hbt

```diff
@@ -81,126 +80,274 @@
+                    {{else}}
+                        <tr>
+                            <td width="250 px">
+                                <strong>{{@key}}</strong>
+                            </td>
+                            <td>{{this}}</td>
+                        </tr>
+                    {{/ifeq}}
+                {{/ifeq}}
+            {{/each}}
+            </tbody>
+        </table>
+    {{/ifeq}}
+{{/each}}
 <h2>Services</h2>
-{{#each Services}}
-    <h3>{{@key}}</h3>
+{{#each Service}}
+    <h3>{{Name}}</h3>
     <table class="table table-striped">
         <tbody>
-        {{#each this.Properties}}
-            <tr>
-                <td width="250 px">
-                    <strong>{{@key}}</strong>
-                </td>
-                <td>{{this}}</td>
-            </tr>
+        {{#each this}}
+            {{#ifeq @key 'ServiceBinding'}}
+            {{else}}
+                <tr>
+                    <td width="250 px">
+                        <strong>{{@key}}</strong>
+                    </td>
+                    <td>{{this}}</td>
+                </tr>
+            {{/ifeq}}
         {{/each}}
         </tbody>
     </table>
-    {{#each this.Bindings}}
-        <h4>{{@key}}</h4>
-        <table class="table table-striped ">
+    {{#each this.ServiceBinding}}
+        <h4>{{Name}}</h4>
+        <table class="table table-striped">
             <tbody>
-            {{#each this}}
+            {{#each this.Slot}}
                 <tr>
                     <td width="250 px">
-                        <strong>{{@key}}</strong>
+                        <strong>{{name}}</strong>
                     </td>
-                    <td>{{this}}</td>
+                    <td>{{join value " "}}</td>
                 </tr>
             {{/each}}
             </tbody>
         </table>
-    {{/each}} {{/each}}
-<h2>Organizations</h2>
-{{#each Organizations}}
-    <h3>{{@key}}</h3>
-    {{#each this.ContactInfo}}
+    {{/each}}
+    {{#each this.Bindings}}
         <h4>{{@key}}</h4>
         <table class="table table-striped ">
             <tbody>
             {{#each this}}
                 <tr>
+                    <td width="250 px">
+                        <strong>{{@key}}</strong>
+                    </td>
                     <td>{{this}}</td>
                 </tr>
             {{/each}}
             </tbody>
         </table>
+    {{/each}} {{/each}}
+<h2>Organizations</h2>
+{{#each Organization}}
+    <h3>{{Name}}</h3>
+    {{#each this}}
+        {{#ifeq @key 'TelephoneNumber'}}
+            <h4>Phone Numbers</h4>
+            {{#each this}}
+                <table class="table table-striped ">
+                    <tr>
+                        <td>{{#if areaCode}}({{areaCode}})
+                        {{/if}}{{#if number}}{{number}}{{/if}}{{#if extension}}
+                            ext. {{extension}}{{/if}}</td>
+                    </tr>
+                </table>
+            {{/each}}
+        {{else}}
```

Comment:
The main reason the "if else if" was added is because there is a case to generate the values dynamically at the end and I did not want duplicate values to be shown.